packages/cloudflare/src/api/get-cloudflare-context.ts

```diff
@@ -29,25 +29,55 @@ const cloudflareContextSymbol = Symbol.for("__cloudflare-context__");
 /**
  * Utility to get the current Cloudflare context
  *
- * Throws an error if the context could not be retrieved
```

Comment:
I was thinking, instead of this we could add a sentence saying something like:
```
During development a local approximation/simulation of the real context is provided
```
or something along those lines

but truthfully I think that devs won't really care that much about this and that it's probably better to keep things simpler here...

what do you think?

Reply:
Simpler is better 👍